packages/app/background-jobs-common/test/processors/TestForeverRescheduledBackgroundJobProcessor.ts

```diff
@@ -21,13 +22,10 @@ export class TestForeverRescheduledBackgroundJobProcessor extends AbstractBackgr
 
   constructor(dependencies: BackgroundJobProcessorDependencies<Data>, redisConfig: RedisConfig) {
     super(dependencies, {
-      queueId: 'TestStalledBackgroundJobProcessor queue',
+      queueId: generateMonotonicUuid(),
       ownerName: 'test',
-      isTest: false, // We don't want to override job options for this processor
```

Comment:
This seems to be a copy-paste from `TestStalledBackgroundJobProcessor`, where values like `stalledInterval` are critical for test performance. 
However in our case, `lockDuration` leads to extra 'Missing lock for job' errors. 
While this doesn’t affect functionality, it adds unnecessary noise. Reducing this noise was part of addressing failing tests, so keeping this change should be beneficial.